Add Documentation for Expert Groups (#3054)
ILIAS-eLearning/ILIAS: docs/development/expert-groups.md

# Experts Groups

**Expert Groups support the ILIAS community in improving quality aspects in the
fields of usability, performance, security, software architecture and the
development process.** These cross cutting concerns are the responsibility of
the Technical Board, as assigned by the by-laws of the ILIAS society. Still,
these fields cannot be treated effectively by the Technical Board alone but
need broad consideration in all stages of the development process. The expert
groups are meant to help the community and the Technical Board by providing
knowledge and insights in various forms.

> To do that, **the groups will work on any means that serve the goal of improved

Reviewer: ```suggestion
To do that, **the groups will work on any means that serve the goal to improve
```

quality e.g. consultancy, tools, code, concepts, documentation, guideline, process
definitions.** Depending on the field, the state of the art in that field and the
problem at hand, different approaches for improvements may be conceivable. The

> experts group are expected to identify and propose means that are effective,

Reviewer: ```suggestion
expert group are expected to identify and propose means that are effective,
```

efficient and appropriate.

To implement them, **the experts use existing processes for decision making, e.g.
the Jour Fixe**, if required. They are also expected to use other processes and
tools provided by the ILIAS community, such as the Mantis Issue Tracker, GitHub,

> the ILIAS conference and development conference. The experts should convince

Reviewer: ```suggestion
the ILIAS conference and the development conference. The experts should convince
```

other community members by tangible reasoning, just as other community members,
and have no special decision making power.

To indicate a special trust of the TB, the **expert groups are appointed by the
Technical Board**. This also gives the experts a special visibility in the
community. **The members of the experts groups are appointed for their well-founded

> knowledge in the respective quality field**. They all are willing and able to

Reviewer: ```suggestion
knowledge in the respective field**. They all are willing and able to
```
Sounds bad and I don't think it is necessary.
